admin-server/go/pkg/file/upload_processor.go

```diff
@@ -241,6 +243,11 @@ func processAndStoreUpload(upload *model.Upload, file *os.File) (uploadProcessRe
 			if util.IsBlankUnicode(cellValue) {
 				numBlankCells++
 			}
+			if len(cellValue) > maxCellSize {
+				tf.Log.Errorw("A cell in your file exceeds the max cell size of 1MB (row : ", i+1, ", column : ", columnIndex+1, ", upload_id : ", upload.ID, ". Please check the file and try again")
+
+				return uploadProcessResult{}, fmt.Errorf("A cell in your file exceeds the max cell size of 1MB (row %v, column %v). Please check the file and try again", i+1, columnIndex+1)
```

Comment:
```suggestion
				return uploadProcessResult{}, fmt.Errorf("A cell in your file exceeds the max cell size of 1MB (row %v, column %v). Please check the file and try again", i+1, columnIndex+1)
```
Line 119 will log this error.